website/content/docs/configuration/session-recording/create-storage-bucket.mdx

```diff
@@ -123,21 +123,18 @@ At this time, the only supported storage is AWS S3.
 
 ### Boundary workers requirements
 
-[Session recording](/boundary/docs/configuration/session-recording) requires at least one PKI worker that:
-- Has access to the AWS S3 storage bucket
-- Has an accessible directory defined by `recording_storage_path` for storing session recordings while they are in progress. On session closure, Boundary moves the local session recording to remote storage and deletes the local copy. For more details, refer to the [PKI worker configuration](/boundary/docs/configuration/worker/pki-worker#session-recording) documentation.
-- Has at least 1 MB of available disk space.
-- Runs Darwin, Windows, or Linux. The following binaries are not supported for session recording: NetBSD, OpenBSD, Solaris.
-
-Development example:
-
-```hcl
-worker {
-auth_storage_path="/boundary/demo-worker-1"
-initial_upstreams = ["10.0.0.1"]
-recording_storage_path="/local/storage/directory"
-}
-```
+[Session recording](/boundary/docs/configuration/session-recording) requires that you configure at least one worker for local storage.
```

Comment:
I'd suggest providing a direct link to the part of the doc that shows how to perform this crucial step

```suggestion
[Session recording](/boundary/docs/configuration/session-recording) requires that you [configure at least one worker for local storage](/boundary/docs/configuration/session-recording/create-storage-bucket#boundary-workers-requirements).
```